nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy

```diff
@@ -86,12 +86,9 @@ class ConanHostedRecipe
   ViewFacet configure(final ConfigurableViewFacet facet) {
     Router.Builder builder = new Router.Builder()
 
-    apiV1.create(builder)
+    addBrowseUnsupportedRoute(builder)
 
-    builder.route(new Route.Builder()
-        .matcher(BrowseUnsupportedHandler.MATCHER)
```

Comment:
Minor: the `BrowseUnsupportedHandler` import at the top of this file can also be removed

Reply:
Thanks, fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/92/commits/522a5008e8f1db2847743259d68d93f144979ee6